example/index.res

```diff
@@ -1,3 +1,39 @@
+module ClickOutside = {
+  @react.component
+  let make = () => {
+    let inputRef = React.useRef(Js.Nullable.null)
+
+    let listRef = React.useRef(Js.Nullable.null)
+
+    let (isClickedOutside, setIsClickedOutside) = React.useState(_ => true)
+
+    let (text, setText) = React.useState(_ => "")
+
+    ClickOutsideHook.use(~refs=[inputRef, listRef], () => setIsClickedOutside(_ => true))
```

Comment:
I wonder, for the sake of keeping the exemple as straightforward as possible, shouldn't we have only 1 ref